test/lib/redis-store-spec.js

```diff
@@ -463,3 +466,39 @@ describe('defaults', function () {
   });
 });
 
+describe('wrap function', function () {
+
+  // Simulate retrieving a user from a database
+  function getUser(id, cb) {
+    setTimeout(function () {
+      cb(null, {id: id, name: 'Bob'});
+    }, 100);
+  }
+
+  var userId = 123;
+  var key = 'user_' + userId;
+  var fromCache = true;
+
+  it('should work with wrap function', function (done) {
```

Comment:
From what I seen it does: he tests that the callback function is executed and that the returned object is the one inserted, and then that the callback function is NOT executed and that the returned object is the one previously inserted (aka stored in store)